exercises/luhn/src/test/java/LuhnValidatorTest.java

```diff
@@ -66,11 +66,14 @@ public void testThatStringContainingPunctuationIsInvalid() {
         assertFalse(luhnValidator.isValid("055-444-285"));
     }
 
+    /* The following test diverges from the canonical test data. This is because the corresponding canonical test does
+     * not account for Java specific functions (such as Character.getNumericValue()), which can be part of incorrect yet
+     * passing implementations. For more detail, check out issue #972 here:
+     * (https://github.com/exercism/java/issues/972).
+    */
     @Ignore("Remove to run test")
     @Test
-    public void testThatStringContainingSymbolsIsInvalid() {
-        assertFalse(luhnValidator.isValid("055£ 444$ 285"));
-    }
+    public void testThatStringContainingSymbolsIsInvalid() { assertFalse(luhnValidator.isValid("34&")); }
```

Comment:
Sorry, I just noticed that this is all on one line. I would put the method body on its own line. It's not very important but it does make it a bit more readable and it's more consistent with the other tests :)